Tests/MeiliSearchIntegrationTests/SearchTests.swift

```diff
@@ -338,6 +338,34 @@ class SearchTests: XCTestCase {
     self.wait(for: [expectation], timeout: TESTS_TIME_OUT)
   }
 
+   // MARK: Crop Marker
+
+  func testSearchCropMarker() {
+    let expectation = XCTestExpectation(description: "Search for Books with a custom crop marker")
+
+    typealias MeiliResult = Result<SearchResult<Book>, Swift.Error>
+    let query = "Manuel"
+    let attributesToCrop = ["comment"]
+    let cropLength = 2
+    let cropMarker = "(ꈍᴗꈍ)"
+    let searchParameters = SearchParameters(query: query, attributesToCrop: attributesToCrop, cropLength: cropLength, cropMarker: cropMarker)
+
+    self.index.search(searchParameters) { (result: MeiliResult) in
+      switch result {
+      case .success(let documents):
+        let book: Book = documents.hits[0]
+        XCTAssertEqual("(ꈍᴗꈍ)Joaquim Manuel(ꈍᴗꈍ)", book.formatted!.comment!)
```

Comment:
>  error: value of optional type 'FormattedBook?' must be unwrapped to refer to member 'comment' of wrapped base type 'FormattedBook'

I could unwrap it more elegantly but since these are tests I'm not sure it is worth the effort.

Reply:
I agree, I said so, because it is a configuration from our linter!